src/pydvl/utils/numeric.py

```diff
@@ -252,7 +253,7 @@ def top_k_value_accuracy(y_true: "NDArray", y_pred: "NDArray", k: int = 3) -> fl
 
 
 def sigmoid(x: float) -> float:
-    return float(1 / (1 + np.exp(-x)))
+    return expit(x).item()
```

Comment:
We depend on scipy for spearman correlation and conjugate gradient, so in principle it's not a problem to add another place where we depend on it. But at the same time, we are unnecessarily increasing the difficulty of getting rid of the dependency at some point (e.g. if we find a better / more suitable CG and throw away the plots with spearman). Since scipy actually does `1/(1+np.exp(-x))` itself (according to the docs), I'd suggest to revert this change.

Reply:
It's more of a general philosophy to reduce dependencies than an optimisation but ok, I'll take those extra 7µs 😄